ethereum/eth/src/test/java/tech/pegasys/pantheon/ethereum/eth/manager/MockExecutorService.java

```diff
@@ -139,7 +140,7 @@ public void execute(final Runnable command) {
     submit(command);
   }
 
-  private static class ExecutorTask<T> {
+  public static class ExecutorTask<T> {
```

Comment:
nit: I don't think this needs to be made public now.

Reply:
Done.